build.sbt

```diff
@@ -43,6 +48,38 @@ lazy val cassandraBundle = (project in file("cassandra-bundle"))
     target in assembly := target.value / "bundle" / "akka" / "persistence" / "cassandra" / "launcher",
     assemblyJarName in assembly := "cassandra-bundle.jar")
 
+// Used for testing events by tag in various environments
+lazy val endToEndExample = (project in file("example"))
+  .dependsOn(core)
+  .settings(libraryDependencies ++= Dependencies.exampleDependencies, publish / skip := true)
+  .settings(
+    dockerBaseImage := "openjdk:8-jre-alpine",
+    dockerCommands :=
+      dockerCommands.value.flatMap {
+        case ExecCmd("ENTRYPOINT", args @ _*) => Seq(Cmd("ENTRYPOINT", args.mkString(" ")))
+        case v                                => Seq(v)
+      },
+    dockerExposedPorts := Seq(8080, 8558, 2552),
+    dockerUsername := Some("kubakka"),
+    dockerUpdateLatest := true,
+    // update if deploying to some where that can't see docker hu
+    //dockerRepository := Some("some-registry"),
+    dockerCommands ++= Seq(
+        Cmd("USER", "root"),
+        Cmd("RUN", "/sbin/apk", "add", "--no-cache", "bash", "bind-tools", "busybox-extras", "curl", "iptables"),
+        Cmd(
+          "RUN",
+          "/sbin/apk",
+          "add",
+          "--no-cache",
+          "jattach",
+          "--repository",
+          "http://dl-cdn.alpinelinux.org/alpine/edge/community/"),
+        Cmd("RUN", "chgrp -R 0 . && chmod -R g=u .")),
+    // Docker image is only for running in k8s
+    javaOptions in Universal ++= Seq("-J-Dconfig.resource=kubernetes.conf"))
```

Comment:
the docs for `-J-D` is somewhat ambiguous but I guess this works
https://www.scala-sbt.org/sbt-native-packager/archetypes/java_app/customize.html#via-build-sbt